trio/_tests/test_ssl.py

```diff
@@ -116,10 +116,8 @@ def ssl_echo_serve_sync(
                     # respond in kind but it's legal for them to have already
                     # gone away.
                     exceptions = (BrokenPipeError, ssl.SSLZeroReturnError)
```

Comment:
Putting these into a tuple only to immediately unpack seems a little redundant.

Reply:
indeed